spyder/plugins/ipythonconsole/scripts/conda-activate.bat

```diff
@@ -10,8 +10,23 @@ set SPYDER_KERNEL_SPEC=%4
 :: Enforce encoding
 chcp 65001>nul
 
-:: Activate kernel environment
-call %CONDA_ACTIVATE_SCRIPT% %CONDA_ENV_PATH%
+IF EXIST "%CONDA_ACTIVATE_SCRIPT%" (
+    :: Activate kernel environment conda, miniconda, mamba
+    call %CONDA_ACTIVATE_SCRIPT% %CONDA_ENV_PATH%
+    goto start
+)
+
+IF DEFINED MAMBA_EXE (
+    :: Activate micromamba environment
+    for /f %%i in ('%MAMBA_EXE% shell activate %CONDA_ENV_PATH%') do set SCRIPT=%%i
+    call %SCRIPT%
+    goto start
+) ELSE (
+    exit 1
+)
```

Comment:
Maybe we could leave this just as `exit 1` since there is a `goto` instructions inside both IF statements?